pkg/products/monitoring/prometheusRules.go

```diff
@@ -299,6 +299,23 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 					},
 				},
 			},
+			{
+				AlertName: "pod-distribution",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "general.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "PodDistribution",
+						Annotations: map[string]string{
+							"sop_url": resources.SopUrlPodDistributionIncorrect,
+							"message": "Pod {{  $labels.namespace  }} / {{  $labels.pod  }} ({{  $labels.container  }}) is incorretly distributed to the zone {{  $value  }} ; for the last 5 minutes",
+						},
+						Expr:   intstr.FromString("kube_pod_info{namespace=~\"redhat-rhoam-.*\", created_by_kind!=\"<none>\"} <= on (namespace, created_by_name) group_left (label_topology_kubernetes_io_zone) (count by (namespace, created_by_name, label_topology_kubernetes_io_zone) (kube_pod_info{namespace=~\"redhat-rhoam-.*\", created_by_kind!=\"<none>\"} == on (node) group_left  (label_topology_kubernetes_io_zone) kube_node_labels) > on (namespace, created_by_name) group_left ceil(count by (namespace, created_by_name) (kube_pod_info{namespace=~\"redhat-rhoam-.*\", created_by_kind!=\"<none>\"} == on (node) group_left  (label_topology_kubernetes_io_zone) kube_node_labels) / scalar(count(count by (label_topology_kubernetes_io_zone) (kube_node_labels)))))"),
```

Comment:
Could we remove the hard-coded ns prefix (`redhat-rhoam`) and use the `nsPrefix` var instead?

Reply:
Updated now. I would love you input on if this is the best place to put this alert or if you think there would be somewhere better?